src/components/ButtonGroupPicker/index.js

```diff
@@ -88,7 +89,7 @@ class ButtonGroupPicker extends Component {
                     <StyledHelpText>{bottomHelpText}</StyledHelpText>
                 </RenderIf>
                 <RenderIf isTrue={!!error}>
-                    <StyledErrorText id={errorMessageId}>{error}</StyledErrorText>
+                    <StyledErrorText id={this.errorMessageId}>{error}</StyledErrorText>
```

Comment:
here we need to use ```this.getErrorMessageId()``` since when error prop is not passed then this id should be undefined